src/main/java/org/opentripplanner/model/Route.java

```diff
@@ -87,6 +92,10 @@ public void setType(int type) {
     }
 
     public String getUrl() {
+        if(url != null) {
+            String cipherText = Crypto.encryptWithExpiry(url, OffsetDateTime.now().plusMinutes(15));
+            return "/otp/redirect/" + cipherText;
```

Comment:
Could we place the endpoint URL where it is more prominent? Where is e.g. /otp defined?

Reply:
You mean moving this into a constant? I can do that.

Reply:
Moved to a constant.